src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp

```diff
@@ -181,6 +181,11 @@ void CompileModelCacheTestBase::run() {
         }
```

Comment:
Such big patch with 0 unit tests is unacceptable.

Reply:
I did not add unit tests that does not use the IE interfaces (`export` and `import`) because serialization needs to be saved to a file and then loaded again to see if it works. And the number of newly enabled functional tests related to serialization is more than 200.

Reply:
Caching functional tests runs the same case three times.

1. run without cache at L200
2. run to create cache at L219 with i =0
3. run with cache at L219 with i =1

Then, it compares the results 1 vs. 2 and 1 vs. 3 at L223

https://github.com/openvinotoolkit/openvino/blob/c51ee524adfd09c9a753fd94d9891626c60878e0/src/tests/functional/plugin/shared/src/behavior/ov_plugin/caching_tests.cpp#L223

But, I agree with you in that we can't say that we can be sure that everything works fine based on them only. I'll add unit tests working on memory as you guided. Thank you.

Reply:
I added 38 unit tests for serialization. These tests have `export_import` in the test case name. Thank you.